src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -306,5 +308,8 @@ public void periodic() {
                 backRightModule.set(states[3].speedMetersPerSecond / MAX_VELOCITY_METERS_PER_SECOND * MAX_VOLTAGE, states[3].angle.getRadians());
 
                 updatePose();
+                SmartDashboard.putNumber("Gyro Roll", getGyroscopeRoll().getDegrees());
+                SmartDashboard.putNumber("Gyro Pitch", getGyroscopePitch().getDegrees());
+                SmartDashboard.putBoolean("Balance Target", getBalanceTarget() != null);
```

Comment:
This might need to be a lambda like ()->, could be why we weren’t seeing it correctly